simplifycommerce.php

```diff
@@ -450,7 +450,7 @@ public function processPayment()
 				'?id_cart='.(int)$this->context->cart->id.'&id_module='.(int)$this->id.'&id_order='.
 				(int)$this->currentOrder.'&key='.$this->context->customer->secure_key, '');
 		else
-			Tools::redirect(Link::getPageLink('order-confirmation.php', null, null,
+			Tools::redirect($this->context->link->getPagelink('order-confirmation.php', null, null,

```

Comment:
Tested all of the scenarios (hosted/regular/saved card) in HTTPS locally and it works